qemu/tests/cfg/balloon_small_size.cfg

```diff
@@ -0,0 +1,12 @@
+- balloon_small_size:
+    virt_test_type = qemu
+    no Win2000
+    only Windows
```

Comment:
Why only Windows here ?  I suggest to use one single case to cover linux and windows guest for the small size test.